### ISSUE TITLE
PackagesNotFoundError

### ISSUE THREAD

**Issue by Dallak** [MontrealCorpusTools/Montreal-Forced-Aligner#270]
Can anyone advise on how to fix the following error? When running the following code:

```
(base) C:\Users\User>conda create -n aligner -c conda-forge openblas python=3.8 openfst pynini ngram baumwelch
```

I got the following errors
```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - baumwelch
  - ngram
  - pynini
  - openfst

Current channels:

  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```
Thanks in advance!

**Comment by mmcauliffe** [comment]
Yes, so those packages are not available on Windows natively.  There are a couple of options, if you need to use G2P functionality, then I would recommend setting up the Windows Subsystem for Linux and following the installation instructions inside of the Linux subsystem.  If you don't need G2P functionality, you can set up the aligner environment without those packages via `conda create -n aligner -c conda-forge openblas python=3.8` and the rest of the functionality should work.

**Comment by Dallak** [comment]
This helped a lot, thank you!

Now when I ran:
```
mfa align test/ librispeech-lexicon.txt english output/
```
I got a different error.

```
All required kaldi binaries were found!
C:\Users\User/Documents/MFA\test\align.log
INFO - Setting up corpus information...
loading from source
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\User\OneDrive - Newcastle University\Documents\MFA\multiprocessing\__init__.py", line 1, in <module>
    from .helper import run_mp, run_non_mp, Stopped, Counter
  File "C:\Users\User\OneDrive - Newcastle University\Documents\MFA\multiprocessing\helper.py", line 6, in <module>
    from ..helper import parse_logs, thirdparty_binary, make_path_safe
ImportError: attempted relative import beyond top-level package
```
Thanks again!

**Comment by mmcauliffe** [comment]
Hm, that path to the python files looks a bit weird.  Did you follow the rest of the installation instructions (https://montreal-forced-aligner.readthedocs.io/en/latest/installation.html) and install MFA via `pip install montreal-forced-aligner`?  It shouldn't get installed to that folder path normally.

**Comment by Dallak** [comment]
It's working now! I just realized that I was working on the wrong path. This happened because there are two paths for the aligner in my computer (i) created manually by simply downloading the zip file of MFA and putting in in Documents- (which is the wrong one), (ii) created by the command `pip install montreal-forced-aligner`. I was using the former path before, but I changed it to the latter, and it is working fine.

Many thanks for your prompt reply!
